Vagrantfile

```diff
@@ -0,0 +1,45 @@
+Vagrant.configure(2) do |config|
```

Comment:
It's not about installing them locally, its about reproducible testing environment. With this one file, everybody who want contribute can get it with ease (without need to install `rvm` to your own machine, I prefer `rbenv` and don't want to mess with it)